CMakeLists.txt

```diff
@@ -103,7 +106,9 @@ if(OMR_GC)
   add_subdirectory("${OMR_GLUE}")
 endif(OMR_GC)
 
-if(OMR_COMPILER)
-  # TODO: Actually support the compiler
-  add_subdirectory(compiler)
-endif(OMR_COMPILER)
+
+
+if(NOT OMR_HOST_OS STREQUAL "win")
```

Comment:
Would be good to leave the `TODO` here to add Windows as its easily grepable. Also slight typo below on "Windows".